pkg/kbld/builder/docker/docker.go

```diff
@@ -101,6 +101,7 @@ func (d Docker) Build(image, directory string, opts DockerBuildOpts) (DockerTmpR
 		cmdArgs = append(cmdArgs, "--tag", tmpRef.AsString(), ".")
 
 		cmd := exec.Command("docker", cmdArgs...)
+		prefixedLogger.Write([]byte(fmt.Sprintf("Invoking docker with args: %s", cmdArgs)))
```

Comment:
@joe-kimmel-vmw dont think we want this in this PR

Reply:
🤦   sorry,  must've had stale changes locally / forgot to reset.  good catch.